### PR TITLE
remove soft colors. add mobile-only and desktop-only classes

### LINKED ISSUE #80: $color-white $color-black
i think we should lose the soft white and black. i think its bad as default

### LINKED ISSUE #82: .mobile-only
should be there by default.  i've seen you use `.only-mobile` but can we do `.mobile-only` instead? @varffs

### PR DESCRIPTION
**Pull request by mikeyrayvon**
replace `$color-black-soft` and `$color-white-soft` with `$color-black` and `$color-white`. close #80 

add `.mobile-only` and `.desktop-only` classes. close #82

### CODE REVIEW

File: css/site.styl [hunk 1]
```diff
@@ -27,21 +27,27 @@ normalize()
...
-  color: $color-black-soft
+  background-color: $color-white
+  color: $color-black
 
 img
   max-width: 100%
   height: auto
 
+.mobile-only
+  display: initial !important
```
Comment (varffs) [inline css/site.styl]: shouldnt need important i dont think

Comment (varffs) [inline css/site.styl]: well i dunno sometimes it 'could' but perhaps we should be making sure not to. I actually don't know why important is considered so evil?

**Comment by varffs** [comment]
i kno they are small but should be 2 diff prs still

File: css/site.styl (hunk 1 above)
Comment (mixcas) [inline css/site.styl]: means u r swiming against the cascade

Comment (varffs) [inline css/site.styl]: but this is a perfect example of needing a class that can override?

Comment (mikeyrayvon) [inline css/site.styl]: i ran into an issue where i was trying to make something mobile-only, but it was being overridden by a utility class on its parent. it was a `ul.u-inline-list li` and i wanted one of the `<li>`s to be mobile-only but it was overridden by that parent class.  i think this is a good use of `! important`

Comment (mixcas) [inline css/site.styl]: yeah, theres cases where is definitely needed, and it's ok. God forbidds.
What I'm saying is use `!important` wisely and ask yourself why do you need it in the first place.

[;

Comment (mikeyrayvon) [inline css/site.styl]: you're right cas. thank you for adding that. i agree with you.

**Comment by mikeyrayvon** [comment]
true 😞  @varffs

**Pull request merged**